jest/helpers/removeRandomizedUrlJestSerializer.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Serializer that removes attribute values that use a google search url.
+ *
+ * We're generating randomized urls for href attributes in links in order to make it easier
+ * to test both visited and unvisited link styles. Because the attributes are randomized,
+ * they'll trigger a snapshot change on every snapshot update. Removing the values entirely
+ * circumvents the issue, and those values weren't important to test anyway.
+ */
+expect.addSnapshotSerializer({
+  test: (value) => {
+    return (
+      typeof value === 'string' &&
+      value.includes('https://www.google.com/search?q=')
+    );
+  },
+  print() {
+    return `"/"`;
+  },
+});
```

Comment:
This looks pretty similar to the Modal serializer. Do you think it would be took awkward to a common utility that would receive `test` and `print` arguments or would that be too close to `addSnapshotSerializer`?

Reply:
Yeah, it's so simple that I kinda feel like we'd just be making an alias for `addSnapshotSerializer`. Though I do find the `addSnapshotSerializer` API really weird so just being able to rename these props might be worth it 😅

Reply:
Yea makes sense that it's so simple it'd just be an alias, upto you but will resolve conversation